changelog/1482.trivial.3.rst

```diff
@@ -0,0 +1,3 @@
+Updated :file:`docs/environment.yml` to use `pip` to install all
```

Comment:
Note that `pip` does not get linked by intersphinx.  I think this is because we don't have `pip` in the mappings, which I think is fine.  You could do one of your fancy replacements to point to the [`pip` documentation](https://pip.pypa.io/en/stable/), or just double back-tick it.